README.md

```diff
@@ -2,6 +2,11 @@
 
 [![Build Status](https://travis-ci.org/alphagov/verify-local-matching-service-example.svg?branch=master)](https://travis-ci.org/alphagov/verify-local-matching-service-example)
```

Comment:
Remove the build status too. (Sorry, should have said.)

Reply:
Have removed the build status :)